src/connectors/intergalacticfm.js

```diff
@@ -1,12 +1,7 @@
 'use strict';
 
-Connector.playerSelector = '#radio_playing_now';
+Connector.playerSelector = '#player';
 
-Connector.artistSelector = '#radio_playing_now > .media-body > h5 > span';
+Connector.artistTrackSelector = '.track-meta > h5';
 
-Connector.trackSelector =
-	'#radio_playing_now > .media-body h5 > span:nth-child(3)';
-
-Connector.isPlaying = () => {
-	return $('#player1').hasClass('jw-state-playing');
-};
+Connector.isPlaying = () => $('.vjs-play-control').hasClass('vjs-playing');
```

Comment:
```suggestion
Connector.isPlaying = () => Util.hasElementClass('.vjs-play-control', 'vjs-playing');
```

We're trying to phase out jQuery from the extension.